docs/SOP.md

```diff
@@ -100,6 +101,20 @@ In general, the metadata record of an ontology in the OBO Foundry metadata regis
 
 For a discussion on this SOP, see [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1848).
 
+<a name="REACTIVATION"></a>
+
+## Reactivating obsolete, orphaned, or inactive ontologies
+
+To mark ontologies that were marked as either obsolete, orphaned, or inactive as active again, we follow this process. 
+Currently, these are the required steps:
+
+1. A member of the development team of the ontology makes a pull request (PR) on the metadata file with the desired change in status. The PR must include evidence to demonstrate new activity, for example by referring to recently-closed issues (we will provide more detailed guidelines as this SOP matures).
+1. If the community member is not the ontology owner, the ontology owner must be contacted for comment via email and tagged on the issue. The ontology owner is given 1 month to object to the change, in which case, it is rejected.
```

Comment:
Delete this part, as it has been incorporated into the previous step.

Reply:
```suggestion
```